autoload/coiledsnake.vim

```diff
@@ -151,14 +151,22 @@ function! coiledsnake#FormatText(foldstart, foldend) abort " {{{1
         let padding = ' ' . repeat(' ', padding)
     endif
 
-    return title . padding . status
+    if g:coiled_snake_explicit_sign_width != 0
+        let rightpadding = g:coiled_snake_explicit_sign_width*2 - 1
+        let rightpadding = ' ' . repeat(' ', rightpadding)
+    else
+        let rightpadding = ''
+    endif
+
+    return title . padding . status . rightpadding
```

Comment:
I don't understand why this code is necessary.  The flags are always meant to be right-justified, so it doesn't make sense to me that there would be right padding.  But maybe I'm missing something.

Reply:
The whitespace ensures that if someone has set `signcolumn` to auto, the text shall not be cut off even if the signcolunm expands. i.e. he/she shall sacrifice right-justification for preservation of information